jetbrains-core/tst/software/aws/toolkits/jetbrains/ui/feedback/FeedbackTest.kt

```diff
@@ -43,9 +43,6 @@ class FeedbackTest {
                 val dialog = FeedbackDialog(projectRule.project)
                 val panel = dialog.getViewForTesting()
 
-                assertThat(panel.sentiment).isEqualTo(Sentiment.POSITIVE)
-
-                panel.clearSentimentSelection()
                 assertThat(panel.sentiment).isEqualTo(null)
```

Comment:
Does posting feedback with it null work, I don't think it would with how the auto-generated code is:
```
        public final Builder sentiment(Sentiment sentiment) {
            this.sentiment(sentiment == null ? null : sentiment.toString());
```

Reply:
No, there is a validation that prevents it from being sent, we just never hit it before:
https://github.com/aws/aws-toolkit-jetbrains/pull/1772/files#diff-4d65ad53aa6a658ac3d9c1983ec36d43R74

Reply:
Ah OK, didn't expect that to exist since we didn't have that scenario before, looks good

Reply:
You. commented on the unit test that validates it 😄